sdk/trace/export.go

```diff
@@ -86,7 +86,7 @@ type SpanData struct {
 	// from StartTime by the duration of the span.
 	EndTime time.Time
 	// The values of Attributes each have type string, bool, or int64.
-	Attributes               map[string]interface{}
+	Attributes               []core.KeyValue
```

Comment:
A late review here, but is the above comment still relevant? I see that core.KeyValue has many different values.

Reply:
it is not relevant. I'll remove in another PR.